artifacts/contracts/discrepancies/nonce/ChainedContracts.sol/ChainedContracts.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
these artifacts should not be committed to the project after this commit https://github.com/hashgraph/hedera-smart-contracts/commit/dfaedf93e5dc857ce2451921c967230099e43ac5. Please kindly rebase main!